include/dsn/dist/cluster_scheduler.h

```diff
@@ -52,7 +52,7 @@ namespace dsn
         DEFINE_THREAD_POOL_CODE(THREAD_POOL_SCHEDULER_LONG)
 
         // ---------- cluster_type -------------
-        enum class cluster_type
+        enum cluster_type
```

Comment:
compilation failures? we intentionally put class here so that any reference needs to be done with a prefix. The enum names right now are too general (e.g., invalid) to be used without a prefix.